cmd/state/main.go

```diff
@@ -251,7 +251,9 @@ func run(args []string, isInteractive bool, cfg *config.Instance, out output.Out
 		if childCmd != nil {
 			cmdName = childCmd.JoinedSubCommandNames() + " "
 		}
-		err = errs.AddTips(err, locale.Tl("err_tip_run_help", "Run → [ACTIONABLE]`state {{.V0}}--help`[/RESET] for general help", cmdName))
+		if !out.Type().IsStructured() {
+			err = errs.AddTips(err, locale.Tl("err_tip_run_help", "Run → [ACTIONABLE]`state {{.V0}}--help`[/RESET] for general help", cmdName))
+		}
```

Comment:
Is there any reasonable way we can avoid the output check? Is this check going to be required for adding other error tips?

Reply:
We only have to do it for this one because we put this tip before all others. Any other tips added (like a link to the forums) comes later and does not need a check like this because `getErrorTips()` will not have those tips yet.

Reply:
Awesome, thank you for the clarification!